index.html

```diff
@@ -157,113 +178,9 @@ <h3 class="title-main">Main Program</h3>
       <section class="main-speakers" title="speakers" id="speakers">
         <h4 class="featured-title title-main">Featured speakers</h4>
         <p class="lowbar-title"></p>
```

Comment:
- [ ] The `Speakers` section looks great, however, you can add the small white and black squares in the back of each picture so this section looks identical to the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230322_091047](https://user-images.githubusercontent.com/25227022/226855615-2ae89600-68ec-4a18-ba3d-0b32cd708a67.png)|![Screenshot_20230322_091102](https://user-images.githubusercontent.com/25227022/226855614-75e8a58e-6f19-493d-983c-b16e33cf29c5.png)|